templates/events-list.php

```diff
@@ -6,41 +6,75 @@
 
 
 
-<h2 class="event_page_title">Upcoming Translation Events</h2>
+<h2 class="event_page_title">Translation Events</h2>
 <div class="event-left-col">
 <?php
-if ( $query->have_posts() ) :
+if ( $current_events_query->have_posts() ) :
 	?>
+	<h3>Current events</h3>
 	<ul>
 		<?php
-		while ( $query->have_posts() ) :
-			$query->the_post();
-			$event_start = get_post_meta( get_the_ID(), '_event_start', true );
+		while ( $current_events_query->have_posts() ) :
+			$current_events_query->the_post();
+			$event_start = ( new DateTime( get_post_meta( get_the_ID(), '_event_start', true ) ) )->format('l, F j, Y');
 			?>
 			<li class="event-list-item">
-				<span class="event-list-date"><time class="event-utc-time" datetime="<?php echo esc_attr( $event_start ); ?>"></span>
-				<a href="<?php echo esc_url( gp_url( wp_make_link_relative( get_the_permalink() ) ) ) ?>"><?php the_title(); ?></a> by <span><?php the_author(); ?></span>
+				<span class="event-list-date"><?php echo esc_html( $event_start ); ?></span>
+				<a href="<?php the_permalink(); ?>"><?php the_title(); ?></a> by <span><?php the_author(); ?></span>
```

Comment:
I'm guessing this change wasn't intended?

```suggestion
				<a href="<?php echo esc_url( gp_url( wp_make_link_relative( get_the_permalink() ) ) ) ?>"><?php the_title(); ?></a> by <span><?php the_author(); ?></span>?></span>
```